pages/pipelines/block_step.md.erb

```diff
@@ -42,6 +42,14 @@ _Optional attributes:_
       A list of input fields required to be filled out before unblocking the step.<br>
       Available input field types: <code>text</code>, <code>select</code>
     </td>
+  </tr>
+    <tr>
+    <td><code>blocked_state</code></td>
+    <td>
+      The state that the build should be set to when the build is blocked by this block step.<br>
+      <em>Default:</em> <code>passed</code><br>
```

Comment:
```suggestion
      <em>Default:</em> <code>nil</code><br>
```

A bit of a technicality in the implementation is that the value defaults to `nil` and Buildkite UI and notifications interpret that as `passed`. If a value isn't set, the user will see `nil` in the API.

Reply:
Hey folks, I'm slightly confused about this one. If set to `passed` then is it like `soft_fail`?

Reply:
Similar, yes. The impact is it will show with the green passed colour on Buildkite. On other connected sites it depends on the fidelity of the integration. For some it shows simply as passed, just like a soft fail. On others it'll show as "Passed and blocked".

Reply:
Perhaps we need a short section about this, or more info in the description?

Reply:
Sounds good to me. @harrietgrace you wanna tackle that ;-) ? Or shall I?